src/main/java/ch/njol/skript/expressions/ExprToggled.java

```diff
@@ -0,0 +1,79 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+
+@Name("Toggled Boolean")
+@Description("The toggled boolean value.")
+@Examples("set {_gravity} to toggled player's gravity")
+@Since("INSERT VERSION")
+public class ExprToggled extends SimpleExpression<Boolean> {
+
+	static {
+		Skript.registerExpression(ExprToggled.class, Boolean.class, ExpressionType.COMBINED, "(toggled|switched) %booleans%");
```

Comment:
I am not really a fan of this pattern. Maybe something like:
```suggestion
		Skript.registerExpression(ExprToggled.class, Boolean.class, ExpressionType.COMBINED, "[the] (inverse|opposite) of %booleans%");
```

You could even make this a property expression, so patterns would be:
```
[the] (inverse|opposite) of %booleans%
%booleans%'[s] (inverse|opposite)
```